app/harvesters/nara_harvester.rb

```diff
@@ -0,0 +1,126 @@
+##
+# A harvester for NARA's API
+#
+# @see Krikri::Harvesters::ApiHarvester
+class NaraHarvester < Krikri::Harvesters::ApiHarvester
+  DEFAULT_URI = 'https://catalog.archives.gov/api/v1'
+  DEFAULT_NAME = 'nara'
+  DEFAULT_BATCHSIZE = 10
+  DEFAULT_ID_FILENAME = '/var/tmp/nara_ids'
```

Comment:
Re the directory location, `/var/tmp` could be scp'd to by anyone with an account (to initially upload and then update the file -- it would work conveniently with just one maintainer, but anyone could do it).  Meanwhile, unless we automated some custom task to create this storage directory and set its permissions, it would be inconvenient to upload to RAILS_ROOT, which is owned by the `dpla` account on our systems.